### PR TITLE
fix: add arpanaditya

### LINKED ISSUE #964: Aditya Arpan Sahoo
## Profile Screenshot

@arpanaditya

### PR DESCRIPTION
**Pull request by arpanaditya**
resolves #964 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
**Hey Arpan, thanks for the contribution. :)**

Please shrink the image (GitHub Profile) to a single page. For reference, you can check some of the close issues [**here**](https://github.com/EddieHubCommunity/awesome-github-profiles/issues?q=is%3Aissue+is%3Aclosed)

File: profiles/arpanaditya.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75975428/149463291-061db5e2-5228-418b-b70f-21ae7f0b75ad.png",
```
Comment (Panquesito7) [inline profiles/arpanaditya.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/149539028-f6ece6db-370b-4b30-adcf-322668957435.png",
```

Comment (arpanaditya) [reply profiles/arpanaditya.json]: Yes, Actually I was thinking about this. But thanks for doing this. :)

**Review by arpanaditya — COMMENTED** [review]
Thank you

**Review by schmelto — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM 🎉, Thanks for the contribution.

**Pull request merged**